dockerfiles/Dockerfile

```diff
@@ -31,6 +31,13 @@ RUN apt-get -y install \
         telnet \
         lsb-release
 
+# Gets the MinIO mc client used to add buckets upon initialization
+# If this client should have issues running inside this image, it is also
+# fine to defer it to a separate image.
+# The current minio/mc Docker image could be a lot smaller
+RUN curl -s -q https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2022-06-11T21-10-36Z -o /usr/bin/mc && \
+    chmod +x /usr/bin/mc
```

Comment:
It seems you included this change in this PR by mistake.

Reply:
Oh yeah, that wasn't supposed to have happened... CTRL-C'ed the git push process too late :)